pkg/auth/user/repository/RoleGroupRepository.go

```diff
@@ -284,3 +306,60 @@ func (impl RoleGroupRepositoryImpl) UpdateRoleGroupIdForRoleGroupMappings(roleId
 	return &model, err
 
 }
+
+func (impl RoleGroupRepositoryImpl) StartATransaction() (*pg.Tx, error) {
+	tx, err := impl.dbConnection.Begin()
+	if err != nil {
+		impl.Logger.Errorw("error in beginning a transaction", "err", err)
+		return nil, err
+	}
+	return tx, nil
+}
+
+func (impl RoleGroupRepositoryImpl) CommitATransaction(tx *pg.Tx) error {
+	err := tx.Commit()
+	if err != nil {
+		impl.Logger.Errorw("error in commiting a transaction", "err", err)
+		return err
+	}
+	return nil
+}
+func (impl RoleGroupRepositoryImpl) GetCasbinNamesById(ids []int32) ([]string, error) {
+	type RoleGroup struct {
```

Comment:
why RoleGroup struct inside the function

Reply:
defined local struct to get only limited data, as gopg considers table name to be by default role_groups ('s) but we have role_group.